debian/postrm

```diff
@@ -15,6 +15,10 @@ remove_cache_dir(){
     rm -rf /var/lib/ubuntu-advantage
 }
 
+remove_logs(){
+    rm -f /var/log/ubuntu-advantage.log*
```

Comment:
Why the asterisk?

Reply:
Log rotation?

Reply:
Globbing there is to catch rotated and compressed or not logs.

Reply:
log rotate right?